Core/Steps/ContinueReleaseStepWithOptionalSupportBranchStepBase.cs

```diff
@@ -41,7 +41,7 @@ protected ContinueReleaseStepWithOptionalSupportBranchStepBase (IGitClient gitCl
   protected void CreateSupportBranchWithHotfixForRelease (SemanticVersion nextHotfixVersion, bool noPush)
   {
     Console.WriteLine("Do you wish to create a new support branch?");
-    if (!InputReader.ReadConfirmation())
```

Comment:
We might want to add the parameter name here since it is not clear what `false` is supposed to do. From the issue description I can guess but just looking at this it is not quite so clear.